libs/libcommon/pyproject.toml

```diff
@@ -16,6 +16,7 @@ huggingface-hub = {extras = ["hf-transfer"], version = "^0.21.4"}
 mongo-types = "0.15.1"
 mongoengine = "^0.27.0"
 networkx = "^3.0"
+numpy = "^1"
```

Comment:
why `numpy = "^1"` here and `numpy = "^1.22.4"` in services/worker?

Reply:
Thanks. I should align both.

Reply:
Done.